src/MongoDB/Driver/BulkWrite.cpp

```diff
@@ -100,82 +100,80 @@ void HHVM_METHOD(MongoDBDriverBulkWrite, update, const Variant &query, const Var
 	bupdate = bson_new();
 	update_converter.convert(bupdate);
 
-	if (!updateOptions.isNull()) {
-		if (options.exists(s_multi)) {
-			Variant v_multi = options[s_multi];
-			bool multi = v_multi.toBoolean();
+	VariantToBsonConverter options_converter(options, HIPPO_BSON_NO_FLAGS);
+	boptions = bson_new();
+	options_converter.convert(boptions);
```

Comment:
If conversion throws, do the `bson_t` variables (e.g. `bquery`, `bupdate`) get cleaned up?